style.css

```diff
@@ -493,53 +508,6 @@ body {
   padding: 20px;
   border-radius: 5%;
 }
-.social-icons {
-  list-style: none;
-  padding: 0;
-  margin-bottom: 0px;
-  z-index: 2;
-}
-
-.social-icons li {
-  display: inline;
-  float: right;
-  margin-right: 25px;
-  margin-top: 30px;
-}
-
-.social-icons a {
-  text-decoration: none;
-  color: white; 
-}
-
-.social-icons i {
-  font-size: 24px; 
-  background-color:   #9795ef; 
-  border-radius:10px; 
-  padding: 10px; 
-  color: white; 
-}
-
-
-
-.circle {
-  width: 190px;
-  height: 190px;
-  border-radius: 50%; 
-  /* display: flex; */
-  justify-content: center;
-  align-items: center;
-  overflow: hidden;
-   margin-top: 0px; 
-  margin-left: 45px; 
```

Comment:
Why is all these removed?

Reply:
Let me check, it wasn't present in my last pull. 
I'll check this again